### ISSUE TITLE
Feature Request: Suggest People In List Edit Mode

### ISSUE THREAD

**Issue by Saijin-Naib** [tuskyapp/Tusky#2768]
Given that people can not be added to lists from their profile, or even by a tap+hold on them anywhere else, it can be hard to remember every person you follow and their handle (or even deal with diacritics or characters not from your language) when editing a list.

I think a possible improvement to this behavior would be to, when the search box on a list edit session is focused, show the people you follow on a scrollable list that will filter as you type.

I think a reasonable first pass at this list would be hiding people already in the list to be edited, prioritizing people not on any list highest, people on another list lowest, and sorting the people in each priority group by recency of follow.

**Comment by sl1txdvd** [comment]
> Given that people can not be added to lists from their profile

As somebody who manually added 80 people to lists yesterday, I think this would have helped me more. It would be nice to have both options.

**Comment by starkej2** [comment]
> Given that people can not be added to lists from their profile, or even by a tap+hold on them anywhere else

Wouldn't it be more straightforward/intuitive to implement adding to lists from the profile view? Similar to how Fedilab works:

![Screenshot_20221107-070654](https://user-images.githubusercontent.com/1320020/200306518-2b6860b6-d716-4d5c-9c7e-7ce6f78aa74a.png)

**Comment by Saijin-Naib** [comment]
Indeed, adding direct from profiles is nice for a single new follow, matches the UX of Twitter, and has Issues open for it already.

This way would be more useful for batch/mass editing of lists, and as others have said, would be a good option to have alongside adding a single user at a time from their profile.

**Comment by SheilaMAverbuch** [comment]
I'd like to up vote adding people to a list from profile view as I've now added about 200 people manually to lists...phew. I'll separately open another issue feature request to add the option to share a list publicly.
